guides/common/modules/proc_configuring-provisioning-callback-for-a-host.adoc

```diff
@@ -47,8 +47,8 @@ For more information about job templates, see http://docs.ansible.com/automation
 |====
 |Name |Value |Description
 |`ansible_tower_provisioning` |true |Enables Provisioning Callback.
-|`ansible_tower_fqdn` |_controller.example.com_ |The fully qualified domain name (FQDN) of your {awx}.
-Do not add `https` because this is appended by {Project}.
+|`ansible_tower_api_url` |_https://controller.example.com/api/controller/v2_ |Defines the fully qualified domain name (FQDN) of your {awx} with the required API path.
```

Comment:
Looking at writers here for the exact phrasing. I mostly care that this is no longer the FQDN but rather a URL (so including a scheme and path).
```suggestion
|`ansible_tower_api_url` |_https://controller.example.com/api/controller/v2_ | Defines the URL of your {awx}, including the required API path.
```